src/Features/Core/Portable/Options/DocumentationCommentOptionsStorage.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.DocumentationComments;
+
+// TODO: move to LSP layer
```

Comment:
Not clear what precisely this means -- is there a tracking item here?

Reply:
We have a general work item to remove IGlobalOptionService from workspace/features layer. That will necessarily result in moving these options to the right place, so this is just a note that these are not in the right place at this point in time.